homeassistant/components/motionmount/select.py

```diff
@@ -23,6 +31,7 @@ async def async_setup_entry(
 class MotionMountPresets(MotionMountEntity, SelectEntity):
     """The presets of a MotionMount."""
 
+    _attr_should_poll = True
```

Comment:
How did they update before?

Reply:
An update was requested as soon as the user selected one of the options. There is always at least the 'wall' preset, so the user could always select something. This wasn't perfect, so I think this improves on auto-detecting new presets and monitoring the connection.